code/modules/atmospherics/machinery/components/binary_devices/pressure_valve.dm

```diff
@@ -64,6 +78,25 @@
 		is_gas_flowing = FALSE
 	update_icon_nopipes()
 
+/obj/machinery/atmospherics/components/binary/pressure_valve/attackby(obj/item/W, mob/user, params)
+	if(istype(W, /obj/item/assembly/spring))
+		if(spring_numbers == max_springs)
+			to_chat(user, "<span class='warning'>There are already [max_springs] connected to the valve!</span>")
+			return TRUE
+		spring_numbers++
+		update_assembly()
+		qdel(W)
+		playsound(get_turf(src), 'sound/items/handling/component_pickup.ogg', 35, TRUE)
+		return TRUE
+
+/obj/machinery/atmospherics/components/binary/pressure_valve/proc/update_assembly()
+	if(spring_numbers > 0)
+		max_output_pressure = MAX_OUTPUT_PRESSURE //reset the pressure to the original one then adds up the new pressure
```

Comment:
how is confusing? one is a define, the other is a var, i can't just put MAX_OUTPUT_PRESSURE or i couldn't change it with the proc

Reply:
I guess, just seems like having it named something else would be more clear. adjusted_max_out_pressure or something. If you think it's fine then leave it, you're way more experienced in DM than me, this is just something that would be annoying in other languages.

Reply:
Suggesting `MAX_OUTPUT_PRESSURE_NORMAL`, this matches the style of  `var/bodytemperature = BODYTEMP_NORMAL`
Otherwise this looks like the max it should go to and never over, not the normal max that can be boosted with parts.

Reply:
changed!